source/include/microapp/cs_MicroappController.h

```diff
@@ -210,6 +235,29 @@ class MicroappController : public EventListener {
 	 */
 	void setScanning(bool scanning);
 
+	/**
+	 * Set the number of empty softInterrupt slots
+	 *
+	 * @param emptySoftInterrupts The new value
+	 */
+	void setEmptySoftInterrupts(uint8_t emptySoftInterrupts);
```

Comment:
The name of this function can be improved, e.g. `setEmptySlotsSoftInterrupts` or something else that describes that this is about the (number of) slots for soft interrupts rather than the interrupts themselves. Same with `incrementEmptySoftInterrupts`.